### ISSUE TITLE
Scene widget icons displayed in wrong order

### ISSUE THREAD

**Issue by qfluxlab** [igorgladkov/rotini#151]
The icons in my scene widget are displayed in the wrong order since the latest update or maybe the one before. I have 0.14.82, OH2. The scene buttons control the correct mapping, but have the wrong icon.

Widget config:

```
Frame label="Media {widget:scene}" {
    Selection item=ST_DI_Harmony_Activity mappings=[
        "PowerOff"="OFF {icon:switch}",
        "HTPC"="HTPC {icon:tv}",
        "Fire TV"="Fire TV {icon:fire}",
        "PS 4"="PS 4 {icon:gamepad}"
    ]
}
```

Displays like this:

![screenshot_2016-07-12-11-32-39](https://cloud.githubusercontent.com/assets/17936469/16775429/261a5780-4826-11e6-9057-b953358f5ade.png)

**Comment by Nebula83** [comment]
Yep, same here. 

```
Frame label="Jasper {widget:scene}" {
  Switch item=swJasperAway  label="[Weg] {icon:door}"
  Switch item=swJasperClose label="[Dichtbij] {icon:car}"
  Switch item=swJasperHome  label="[Thuis] {icon:home}"
}
Frame label="Marloes {widget:scene}" {
  Switch item=swMarloesAway  label="[Weg] {icon:door}"
  Switch item=swMarloesClose label="[Dichtbij] {icon:car}"
  Switch item=swMarloesHome  label="[Thuis] {icon:home}"
}
```

![screenshot_2016-07-13-07-01-25 - copy](https://cloud.githubusercontent.com/assets/12481964/16793380/eb44f344-48d0-11e6-8be6-5de0c83322ac.png)

It appears to be semi-random as well.

**Comment by igorgladkov** [comment]
Fixed in 0.17
